main.js

```diff
@@ -0,0 +1,64 @@
+let library = [];
+
+const getInput = () => {
+  const book = {};
+  const titleInput = document.getElementById('bookTitle');
+  const authorInput = document.getElementById('bookAuthor');
+
+  // Return null if either title or author input is empty
+  if (titleInput.value.trim() === '' || authorInput.value.trim() === '') {
+    return null;
+  }
+
+  book.title = titleInput.value;
+  book.author = authorInput.value;
+  return book;
+};
+
+const removeBook = (title) => {
+  const book = document.getElementById(title);
+  book.remove();
+  library = library.filter((bookObj) => bookObj.title !== title);
+  localStorage.setItem('library', JSON.stringify(library));
+};
+
```

Comment:
- [ ] You remove the boos based on the book's **_title_**. This is not unique enough because two books with the same title will be removed at the same time when a user clicks the remove button on one of them. I would advise that you check for uniqueness if you still want to use the _**title**_ to remove a book. 

A better way to implement this would be to create a unique `ID` for each book as they are added and then you can remove a book using the `ID`. Kindly implement the logic that suits you better but please ensure that the correct book is removed even if there is more than one book that has the same title or author. 👍 💻